cli/config/compose/services/debian-systemd/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2.3'
+services:
+  debian-systemd:
+    image: alehaa/debian-systemd:${debian_systemdTag:-stretch}
```

Comment:
Should we store our custom version of this image? 

https://github.com/alehaa/docker-debian-systemd